smartseq2_single_sample/check_reference/tasks/featurecounts.wdl

```diff
@@ -0,0 +1,46 @@
+task FeatureCountsUniqueMapping {
+  File aligned_bam
+  File gtf
+  String fc_out
+  
+  command {
+    featureCounts -T 4  -s 0 -t exon -g gene_id -p -B -C -a ${gtf} -o "${fc_out}.gene.unq.counts.txt" ${aligned_bam}
```

Comment:
Marcus tells me it is generally good practice to double quote all variable expansions. His evidence seemed compelling: http://mywiki.wooledge.org/Quotes